src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const MIDDLE_TERM = 3;
+  const LONG_TERM_DISCOUNT = 50;
+  const MIDDLE_TERM_DISCOUNT = 20;
+  const PRICE_PER_DAY = 40;
+
+  if (days >= LONG_TERM) {
+    return PRICE_PER_DAY * days - LONG_TERM_DISCOUNT;
+  }
+
+  if (days >= MIDDLE_TERM) {
+    return PRICE_PER_DAY * days - MIDDLE_TERM_DISCOUNT;
+  }
+
+  return PRICE_PER_DAY * days;
```

Comment:
You can create a variable for base price and use it everywhere
```suggestion
  const PRICE_PER_DAY = 40;
  const BASE_PRICE = PRICE_PER_DAY * days;

  if (days >= LONG_TERM) {
    return BASE_PRICE - LONG_TERM_DISCOUNT;
  }

  if (days >= MIDDLE_TERM) {
    return BASE_PRICE - MIDDLE_TERM_DISCOUNT;
  }

  return BASE_PRICE;
```